src/components/Pressable/GenericPressable/index.js

```diff
@@ -0,0 +1,35 @@
+import React, {forwardRef} from 'react';
+import _ from 'lodash';
+import GenericPressable from './GenericPressable';
+import GenericPressablePropTypes from './PropTypes';
+
+const WebGenericPressable = forwardRef((props, ref) => {
+    // change all props with accessibility* to aria-*
+    const accessibilityMappedProps = _.mapKeys(props, (value, key) => {
```

Comment:
Clever solution – I just wish we didn't have to do it at runtime. I bet we could contrive some webpack magic to do this conversion for us at compile time 🤔

Reply:
I'm not sure about doing it at compile time, as in this scenario we are mapping passed props, on compile time we won't know what are those and moreover props can change while runtime.

I'm thinking about replacing whole `mapKeys` approach with more strightforward solution - redirecting props to different keys, sth like:
`<Pressable id={props.nativeID}  aria-label={props.accessibilityLabel}  {...rest} />`

It will be easier to read, but will need to manually map all relevant accessibility props

Reply:
Yeah, while it's more verbose I think that's ultimately a bit simpler, so I like that change